app/lib/specialist_document_renderer.rb

```diff
@@ -0,0 +1,19 @@
+require "markdown_attachment_processor"
+require "specialist_document_header_extractor"
+require "govspeak_to_html_renderer"
+
+class SpecialistDocumentRenderer
+  def self.create
+    ->(doc) {
+      pipeline = [
+        MarkdownAttachmentProcessor.method(:new),
+        SpecialistDocumentHeaderExtractor.create,
+        GovspeakToHTMLRenderer.create,
+      ]
```

Comment:
It feels like it might be a bug that this and the `ManualDocumentRenderer` you deal with in the next few commits are subtly different; this one doesn't do `FootnoteSomethingSomething` in its pipeline, but the `ManualDocumentRenderer` does.  We should check obviously, but I think we can get rid of this one and replace it's single use with an instance of the `ManualDocumentRenderer` instead.